packages/typespec-azure-resource-manager/test/resource.test.ts

```diff
@@ -823,68 +672,41 @@ interface RestorePointOperations {
 
     `);
 
-    const resources = getArmResources(program);
-    expectDiagnosticEmpty(diagnostics);
```

Comment:
I notice the expectDiagnosticEMpty being removed in some of these tests, is this because of the removal of docs, or does the new tester automatically check for this so it's no longer necessary?

Reply:
Nothing actually changed here with the new tester but if you use .compile it automatically validate no failures which I changed to use that instead of compileAndDiagnose